test/component_resolver_test.rb

```diff
@@ -26,4 +29,15 @@
       assert_match /<test-govuk-component data-template="govuk_component-name">/, templates.first.args[0]
     end
   end
+
+private
```

Comment:
I don't think access modifiers are necessary in tests. If it's to control scoping for other methods that would clash, that's a larger issue that should be addressed.